spanner/spec/spec_helper.rb

```diff
@@ -16,6 +16,7 @@
 require "google/cloud/spanner"
 require "google/cloud/spanner/admin/instance"
 require "google/cloud/spanner/admin/database"
+require_relative "../spanner_samples"
```

Comment:
[Nit] This is a bit worrying, since it creates a dependency in the wrong direction :) But I see the helpers are using the samples, so it is necessary. We might, however, end up with circular dependency issues in the future.

Reply:
ya that was a bit cumbersome, but should be fine as we just require the helper in tests.